scripts/runtests.py

```diff
@@ -89,21 +92,45 @@ def main() -> None:
     print("\nRunning check_new_syntax.py...")
     check_new_syntax_result = subprocess.run([sys.executable, "tests/check_new_syntax.py"])
 
-    print(f"\nRunning Pyright on Python {_PYTHON_VERSION}...")
+    strict_params = _get_strict_params(path)
+    print(f"\nRunning Pyright ({'stricter' if strict_params else 'base' } configs) for Python {_PYTHON_VERSION}...")
     pyright_result = subprocess.run(
-        [sys.executable, "tests/pyright_test.py", path, "--pythonversion", _PYTHON_VERSION] + _get_strict_params(path),
+        [sys.executable, "tests/pyright_test.py", path, "--pythonversion", _PYTHON_VERSION] + strict_params,
         stderr=subprocess.PIPE,
         text=True,
     )
-    if re.match(r"error (runn|find)ing npx", pyright_result.stderr):
-        print(colored("\nSkipping Pyright tests: npx is not installed or can't be run!", "yellow"))
+    if re.match(_NPX_ERROR_PATTERN, pyright_result.stderr):
+        print(_NPX_ERROR_MESSAGE)
         pyright_returncode = 0
         pyright_skipped = True
     else:
         print(pyright_result.stderr)
         pyright_returncode = pyright_result.returncode
         pyright_skipped = False
 
+    print(f"\nRunning Pyright regression tests for Python {_PYTHON_VERSION}...")
+    pyright_testcases_result = subprocess.run(
+        [
+            sys.executable,
+            "tests/pyright_test.py",
+            Path(path) / "@tests",
+            "--pythonversion",
+            _PYTHON_VERSION,
+            "-p",
+            _TESTS_CONFIG_FILE,
+        ],
+        stderr=subprocess.PIPE,
```

Comment:
(No functional difference, but I find this more readable)

```suggestion
        capture_output=True,
```

Reply:
The output no longer appears in the console. And that was in line with the rest of the tests in this file. I've reverted it.